### PR TITLE
Stop loop

### PR DESCRIPTION
**Pull request by shoukoo**
I have tested the changes successfully on my local and cron server

### CODE REVIEW

**Comment by shoukoo** [comment]
@alexlance , can you please review it again?

File: ops/cron [hunk 1]
```diff
@@ -0,0 +1,3 @@
+MAILTO=""
+# run at 11 am everyday
```
Comment (alexlance) [inline ops/cron]: comment is not quite right

File: ops/cron [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+*/5 * * * * root . /root/.env; CRONLOCK_KEY="newrelic-plugin" cronlock bash -c "cd /usr/local/newrelic_aws_cloudwatch_plugin-latest && /usr/local/bin/bundle exec bin/newrelic_aws"
```
Comment (alexlance) [inline ops/cron]: you should be able to remove the `cd /usr/local...` bit, because the WORKDIR is set in the Dockerfile.

File: ops/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euxo pipefail
+
+
+docker build -t docker.camplexer.com/newrelic-plugin:latest .
+docker push docker.camplexer.com/newrelic-plugin:latest
+image="newrelic-plugin:$(git log -1 --format='%h')"
+docker tag docker.camplexer.com/newrelic-plugin:latest docker.camplexer.com/${image}
+docker push docker.camplexer.com/${image}
+
```
Comment (alexlance) [inline ops/deploy.sh]: I'm not sure about this script. There should probably be a build.sh and then a deploy.sh.

But if we're going to bundle it all up together for now - then the 5 lines above should look more like:
```
image="newrelic-plugin:$(git log -1 --format='%h')"
docker build -t docker.camplexer.com/${image}
docker push docker.camplexer.com/${image}
```
So, avoid using latest tag.

File: ops/cron (hunk 2 above)
Comment (shoukoo) [reply ops/cron]: that didn't work for me, the system kept one telling me cannot find Gemfile.

File: ops/deploy.sh (hunk 3 above)
Comment (shoukoo) [reply ops/deploy.sh]: OK changed, and only master branch can run this script.

**Pull request merged**